src/flag_gems/runtime/__init__.py

```diff
@@ -0,0 +1,21 @@
+from . import backend, commom_utils, register
+from .backend.device import DeviceDetector
+from .configloader import ConfigLoader
+
+configer = ConfigLoader()
```

Comment:
the name 'configer' is a bit weird. just name it config_loader or something else.

Reply:
Done.